query-engine/driver-adapters/src/conversion.rs

```diff
@@ -31,6 +32,10 @@ impl ToNapiValue for JSArg {
         match value {
             JSArg::RawString(s) => ToNapiValue::to_napi_value(env, s),
             JSArg::Value(v) => ToNapiValue::to_napi_value(env, v),
+            JSArg::Buffer(bytes) => ToNapiValue::to_napi_value(
+                env,
+                napi::Env::from_raw(env).create_arraybuffer_with_data(bytes)?.into_raw(),
```

Comment:
TODO: test `Bytes` support in other adapters.

For libSQL it's essential that we pass these as `ArrayBuffer`s as we don't have sufficient type information after this point in general case.

If any other adapters require base64-encoded strings, either they will need to communicate their preference similar to how they report the `flavour`, or additional mapping needs to happen on JS side.